pages/public_cloud/data_analytics/analytics/information_05_capabilities/guide.en-gb.md

```diff
@@ -0,0 +1,81 @@
+---
+title: Data Analytics - Capabilities and Limitations
+excerpt: Discover the capabilities and limitations of Data Analytics
+updated: 2025-01-07
+---
+
+## Data Analytics
+
+With our data analytics services, we take care of the database infrastructure and administration, including set-up, backup, scalability, and security.
```

Comment:
With our analytics services, we take care of the infrastructure and administration, including set-up, backup, scalability, and security.